src/content/en/2020/javascript.md

```diff
@@ -606,7 +608,7 @@ We get a very similar picture when looking at main thread time for pages where t
   sql_file="main_thread_time_frameworks.sql"
 ) }}
 
-Ember's mobile main thread time jumps out and kind of distorts the graph with how long it takes. Pulling it out makes the picture a bit easier to understand.
+Ember's mobile main thread time jumps out and kind of distorts the graph with how long it takes. I spent some more time looking into this and it appears to be heavily influence [by one particular platform using this framework inefficiently](https://timkadlec.com/remembers/2021-01-26-what-about-ember/), rather than an underlying problem with Ember itself. Pulling it out makes the picture a bit easier to understand.
```

Comment:
@tkadlec know you were going to raise a PR for this, but had another issue raised to me in this chapter, and spotted some other things, so thought I'd include this too.

I threw some wording in here, but feel free to change wording or changing to a note instead as discussed, in this branch.

@noah-vdv once Tim approves or changes this would you mind changing the Dutch version? I auto-translated above suggestion but am sure it's far from perfect Dutch.

Reply:
Made a tiny tweak (Added parenthesis and a 'd' to 'influence')

LGTM. Thanks for doing this @bazzadp. I'll owe you several beers whenever we finally get to hang out in person again. :)

Reply:
Oh good spot!